content/en/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -99,7 +99,9 @@ NamespaceLifecycle, LimitRanger, ServiceAccount, TaintNodesByCondition, Priority
 
 ## What does each admission controller do?
 
-### AlwaysAdmit {#alwaysadmit} {{< feature-state for_k8s_version="v1.13" state="deprecated" >}}
+### AlwaysAdmit {#alwaysadmit}
```

Comment:
nit: Just checking, are the named anchors needed?
I expect these are automatically created and that the links were correct before the theme was introduced.

Reply:
They're used by links in other parts of the docs –  I'd leave them in for now.

Reply:
Right. I don't mind adding the named anchors if the names are different from what is automatically generated or the automatically generated name is too long. The other reason is if the heading changes and you want to preserve the link.